interface/src/interfaces/Game.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <!-- <BasicTopBottom
+        @arrow-up-down="arrowUpDown"
+        @arrow-up-up="arrowUpUp"
+        @arrow-down-down="arrowDownDown"
+        @arrow-down-up="arrowDownUp" /> -->
+    <BasicLeftRight
+        @arrow-left-down="arrowLeftDown"
+        @arrow-left-up="arrowLeftUp"
+        @arrow-right-down="arrowRightDown"
+        @arrow-right-up="arrowRightUp" />
+</template>
+
+<script>
+// import BasicTopBottom from './controllers/basic-top-bottom/BasicTopBottom.vue'
+import BasicLeftRight from './controllers/basic-left-right/BasicLeftRight.vue'
+import { ARROW_DOWN, ARROW_LEFT, ARROW_RIGHT, ARROW_UP, KEY_DOWN, KEY_UP } from './Keys'
+
+export default {
+  name: 'Game',
+  components: {
+    // BasicTopBottom,
+    BasicLeftRight
+  },
+  methods: {
+    arrowUpDown () {
+      this.$store.dispatch('pressButton', { key: ARROW_UP, keyState: KEY_DOWN })
```

Comment:
ten pressButton to magic string, stałka powtarzająca się w kilku miejscach. Proponuję zapisać to do jakiejś ogólnej stałej i tej stałej używać w miejscu wystąpienia tego stringa. Będzie łatwiej, gdybyśmy kiedys potrzebowali zmienić nazwę eventu